wallet/exec/Main.hs

```diff
@@ -54,4 +57,10 @@ initializeEverything serverAddress = do
         uiAction :: IO ()
         uiAction = mkUiAction (knitFaceToUI uiWalletState uiFace walletFace knitFace)
 
-    return uiAction
+        refreshAction :: IO ()
+        refreshAction = forever $ do
+            delay <- randomRIO (15, 45)
```

Comment:
Slot duration in production will be 20s, I believe we can make this less.
Poor @ArgentumHeart may just fall asleep while waiting for 40 seconds.

Reply:
15..25 then